css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: "oppins", sans-serif;
+  background-image: url("../img/background.png");
+  background-repeat: no-repeat;
+}
+
+.container {
+  margin: 5%;
+  padding-top: 3rem;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1em 1.8em;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 1.2rem;
+  line-height: 1.4rem;
+  text-align: center;
+  color: #6070ff;
+}
+
+.bars {
+  color: #6070ff;
+}
+
+.hero-main-text {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2rem;
+  color: #172b4d;
+}
+
+.hero-minor-text {
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.8rem;
+}
+
+.text-connect {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 1.2rem;
+  color: #7f8cff;
+}
+
+.icons-hero {
+  color: #505f79;
+  font-size: 0.8rem;
+}
+
+.icons-hero span {
+  padding-right: 1rem;
+}
+
+/* @media only screen and (max-width: 375px) {
+}
+
+@media only screen and (min-width: 376px) and (max-width: 768px) {
+} */
```

Comment:
- [ ] Kindly remove this commented-out code. removing such codes make your work look clean 👍

Reply:
done :heavy_check_mark: